src/storage/kv/transaction.rs

```diff
@@ -439,7 +439,7 @@ impl Transaction {
     }
 
     /// Commits the transaction, by writing all pending entries to the store.
-    pub async fn commit(&mut self) -> Result<()> {
+    pub async fn commit(&mut self) -> Result<Option<(u64, u64)>> {
```

Comment:
I'm not sure that exposing internal internal details like txid and commit_ts in a public API is a good idea.

Reply:
Noted, made it private to keep this api unchanged